src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(pyxtp LANGUAGES CXX)
+
+find_package(pybind11 REQUIRED HINTS)
+find_package(VOTCA_XTP REQUIRED NO_MODULE)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds
+ PUBLIC
+ ${PROJECT_SOURCE_DIR}/binds/include
+)
+
+target_compile_options(xtp_binds
+ PUBLIC
+ "-Wall"
+)
+set_target_properties(xtp_binds
+  PROPERTIES
+  CXX_STANDARD 14
+  CXX_STANDARD_REQUIRED ON
+)
+
+
+target_link_libraries(xtp_binds
+ PUBLIC
+ VOTCA::votca_xtp VOTCA::votca_tools
```

Comment:
`VOTCA::votca_tools` can be dropped as xtp pull that in, plus you are not searching for it.